### DISCUSSION TITLE
Cannot access any of my projects w/ the dashboard

### DISCUSSION THREAD

**Discussion by mightymitegames** [supabase/supabase#19505] [Questions] [answered]
Hi all -- since yesterday I have been unable to access any of my projects in the dashboard. The Docs area comes up fine, but anything that would show specific info on one of my projects simply doesn't appear on the screen. I can't even send a message to support because that won't go through either. I have tried signing off and back on, and clearing my browser cache.

**Comment by ssudaraka** [comment, marked as answer]
Few suggestions that you could try.
- Try a different browser and see if that resolves the issue.
- Check your machine's time is correct. You can visit https://time.is/ to check if your machine's clock is correct.
- Check if you have a pending browser upgrade. Retry after upgrading your browser to the latest version.

If the above doesn't help, please open a support ticket via support@supabase.com. When you send the support ticket, make sure you mention what you tried to fix the issue :)

**Reply by mightymitegames** [reply]
Thanks!  Switching to Firefox did the trick (the machine clock was fine though and I had no pending browser updates to Chrome). Any thoughts on how I might get things working in Chrome again? I also cleared out cookies and local storage on Chrome and nothing seems to help there.
